openedx/adg/lms/webinars/helpers.py

```diff
@@ -234,37 +235,47 @@ def remove_emails_duplicate_in_other_list(email_list, reference_email_list):
     return [email for email in email_list if email not in reference_email_list]
 
 
-def update_webinar_team_registrations(webinar_form):
+def get_newly_added_and_removed_team_members(webinar_form):
```

Comment:
Can we rename this to `get_newly_added_and_newly_removed_team_members` as currently it is a little confusing.

Reply:
I would not have brought it up if it wasn't confusing for me. However, if you think it won't improve the readability then this is fine too.